### DISCUSSION TITLE
[Webapp/Mautic] Incorporate Mautic 5 release

### DISCUSSION THREAD

**Discussion by DerLeole** [shinsenter/php#39] [Ideas]
A couple of days ago Mautic 5 was finally released to the public.

I know its probably not a priority, but it would be nice to have the included script for mautic be changed to install Mautic 5 going forward. Especially since their own docker images are not updated yet and yours offer compose support, which is more emphasised in Mautic 5.

https://www.mautic.org/blog/community/mautic-5-beyond-expectations-beyond-limits

Thank you very much!

**Comment by shinsenter** [comment]
Hey @DerLeole 

I'm working on new docker containers that support Mautic 5.

**Reply by DerLeole** [reply]
Awesome!

**Reply by shinsenter** [reply]
@DerLeole 
We have released totally new Docker images for Mautic v5.
https://hub.docker.com/r/shinsenter/mautic

Currently, Mautic v5 only supports PHP 8.0 and 8.1, and I hope they can soon provide support for newer PHP versions.

**Reply by DerLeole** [reply]
Thank you very much!

I will test this out shortly.

Does this mean old Mautic 4 images are totally unavailable or have they simply been moved to the -deprecated tags?

**Reply by shinsenter** [reply]
@DerLeole 

The new Mautic docker images using PHP >= 8.0 will support running Mautic v5 and above by default. They will also initialize a new Mautic v5 project when you mount the application directory into an empty folder.

But this does not mean you can only use them for Mautic v5. You can mount a folder containing the source code of an existing Mautic v4.4 installation (requiring PHP 8.0).
View supported PHP versions by each Mautic version here:
https://www.mautic.org/download/requirements

To run older versions of Mautic, you can directly use other variants like [shinsenter/phpfpm-nginx](https://hub.docker.com/r/shinsenter/phpfpm-nginx) or [shinsenter/phpfpm-apache](https://hub.docker.com/r/shinsenter/phpfpm-apache) and mount the directory containing the Mautic source code. 

Example Dockerfile to build your own Mautic v4 container:
```Dockerfile
FROM shinsenter/phpfpm-apache:php7.4-tidy

ENV DOCUMENT_ROOT="/docroot"
ENV INITIAL_PROJECT="mautic/recommended-project:^4"
ENV DISABLE_GENERATING_INDEX=1
ENV COMPOSER_REQUIRES_GIT=1
```

You can also choose to use the older docker image versions (tagged with `*-deprecated`), but we have stopped updating those docker images.
